src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -11,6 +11,9 @@ class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
     |  output {
     |    File procs = stdout()
     |  }
```

Comment:
Hmm, what do you think about creating a new test, like `RuntimeSectionSpec` where you have a minimum WDL that expresses runtime... something like

```
task test {
  command { ps }
  runtime { "docker": "ubuntu/latest" }
}

workflow wf { call test }
```

This way each test's WDL file can encompass only the thing it's testing.